Content.Shared/Slippery/SlipperySystem.cs

```diff
@@ -72,6 +74,9 @@ private void TrySlip(EntityUid uid, SlipperyComponent component, EntityUid other
         if (HasComp<KnockedDownComponent>(other) && !component.SuperSlippery)
             return;
 
+        if (!_random.Prob(component.SlipProbability))
+            return;
```

Comment:
Won't this cause prediction issues since it rolls on both client and server and could get different results?

Reply:
yeah, youre right

Reply:
on client if the prob is not 1 it should return immediately
on server it should send an event or something to tell clients that the slip roll succeeded